### ISSUE TITLE
fix Node memory limit

### ISSUE THREAD

**Issue by siem4** [xeokit/xeokit-convert#180]
I have made a previous issue detailing this issue. #177 

TL;DR; node memory limit is reached and this is in the pako package. 

Currently, there is a string parsed to this function with all the metadata. To fix this, pako should receive a buffer. I have made the fix, and it works for me.
![afbeelding](https://github.com/user-attachments/assets/45775cd6-1117-4219-8540-1629cc1b964f)

the pako docs https://nodeca.github.io/pako/#deflate

This is done in the writeXKTModelToArrayBuffer.js on line 456.  
The model used for this test is https://github.com/buildingSMART/Sample-Test-Files/tree/master/IFC%202x3/SDK%20-%20S1 and only the `bim bouwkundig.ifc` file.

In some way, Pako does not handle the 127 MB input string well. With a buffer, this problem is resolved. Due to better memory management of buffers.

**Comment by siem4** [comment]
In some way, Pako does not handle the 127 MB string input well. With a buffer, this problem is resolved.

**Comment by siem4** [comment]
@xeolabs is there a possibility this could get fixed?

**Comment by xeolabs** [comment]
If I understand correctly, there's nothing we can fix here, since it appears to be a Pako issue.

I recommend:

- Use convert2xkt with -u parameter, which will disable Pako zipping

and

- Use the conversion pipeline documented [here](https://xeokit.github.io/xeokit-model-conversion-tests/ifc-tests.html), which splits the IFC into a set of XKT files, with metadata being held in external JSON files
- As described in those docs - don't embed the metadata JSON in the XKT, keep it external
